Source/Core/Core/PowerPC/Jit64/Jit_Branch.cpp

```diff
@@ -90,11 +91,12 @@ void Jit64::bx(UGeckoInstruction inst)
 #endif
   if (destination == js.compilerPC)
   {
-    // PanicAlert("Idle loop detected at %08x", destination);
-    // CALL(ProtectFunction(&CoreTiming::Idle, 0));
-    // JMP(Asm::testExceptions, true);
-    // make idle loops go faster
-    js.downcountAmount += 8;
+    ABI_PushRegistersAndAdjustStack({}, 0);
+    ABI_CallFunction(CoreTiming::Idle);
+    ABI_PopRegistersAndAdjustStack({}, 0);
+    MOV(32, PPCSTATE(pc), Imm32(destination));
```

Comment:
Is it necessary to update PC here? WriteExceptionExit() copies PC to NPC, but doesn't seem to do anything to PC itself? I'm not familiar with how the jit updates the PC register in relation to the instructions as they are translated, but I assume it doesn't happen every instruction.